README.md

```diff
@@ -30,78 +28,37 @@ npm install -g @stoplight/spectral
 yarn global add @stoplight/spectral
 ```
 
-Spectral is avaiable as a Docker image as well
-
-```bash
-docker run --rm -it stoplight/spectral lint "${url}"`
-```
-
-If the file you want to lint is on your computer, you'll need to mount the directory where the file resides as a volume
-
-```bash
-docker run --rm -it -v $(pwd):/tmp stoplight/spectral lint "/tmp/file.yaml"
-```
-
-Finally, [executable binaries](#executable-binaries) are also available.
-
-## Usage
-
-### CLI
-
-After installing via one of the methods above, Spectral can be used via your command-line. Take a look at the [CLI docs](./docs/cli.md) for information on the options available.
+For more installation options, see [Getting Started > Installation](https://stoplight.io/p/docs/gh/stoplightio/spectral/docs/getting-started/installation.md)
 
-### Examples
+## Getting Started
 
-- [Adding a rule](./docs/rulesets.md#adding-a-rule)
-- [Adding to the recommended OpenAPI rules](./docs/rulesets.md#adding-to-the-recommended-openapi-rules)
-- [Enabling specific OpenAPI rules](./docs/rulesets.md#enabling-specific-openapi-rules)
-- [Disabling specific OpenAPI rules](./docs/rulesets.md#disabling-specific-openapi-rules)
-- [Changing the severity of a rule](./docs/rulesets.md#changing-the-severity-of-a-rule)
+After [installation](https://stoplight.io/p/docs/gh/stoplightio/spectral/docs/getting-started/installation.md) take a look at our [getting started documentation](https://stoplight.io/p/docs/gh/stoplightio/spectral/docs/getting-started/concepts.md).
 
-## Concepts
-
-There are three key concepts in Spectral: **Rulesets**, **Rules**, and **Functions**.
-
-- **Rulesets** act as a container for rules and functions.
-- **Rules** filter your object down to a set of target values, and specify the function that is used to evaluate those values.
-- **Functions** accept a value and return issue(s) if the value is incorrect.
-
-Think of a **Spectral ruleset** as a flexible and customizable style guide for your JSON objects.
-
-## Programmatic usage
-
-Spectral is written in TypeScript (then compiled to JavaScript) and can be used directly for when you need to use Spectral programmatically. Take a look at our ["JavaScript API documentation"](docs/js-api.md).
+- [Adding a rule](https://stoplight.io/p/docs/gh/stoplightio/spectral/docs/getting-started/rulesets.md#adding-a-rule)
```

Comment:
```suggestion
- [Adding a rule](docs/getting-started/rulesets.md#adding-a-rule)
```

Reply:
Seeing as the README is not going to be read by platform these links will not be transformed to docs URLs and we'll be sending our GitHub users to the markdown files here in the repo. This is temporary until we get symlinks supported in Studio, and can symlink the README.md to docs/intro.md.